package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "channelape-sdk",
-  "version": "1.24.0",
+  "version": "1.25.0-develop.0",
```

Comment:
This should be 1.26.0-develop.0

Reply:
@ryankazokas There's already a 1.26.0-develop.0, and I just released 1.26.0 to master.  This will need to be 1.27.0

Reply:
Thanks @mb-ca . @ShannonTalarico sorry about that. Update version to 1.27.0-develop.0

Reply:
@ShannonTalarico i'll fix this during merge, but you should have updated the version in package.json and not package-lock.json. Package-lock.json is generated during npm install.